src/test/java/org/opentripplanner/raptor/moduletests/C01_TransferBoardAndAlightSlackTest.java

```diff
@@ -74,18 +75,22 @@ void setup() {
   }
 
   static List<RaptorModuleTestCase> testCases() {
+    // TODO expMinDuration= walk 50s + Transit 1m20s + 1x transfer 1m + 2x board/alight slack 1m20s
+    //      = 4m30s
+
     var expected =
-      "Walk 30s ~ B ~ " +
-      "BUS R1 0:02:11 0:03:01 ~ C ~ " +
-      "BUS R2 0:04:41 0:05:01 ~ D ~ " +
-      "Walk 20s " +
+      "Walk 30s ~ B " +
+      "~ BUS R1 0:02:11 0:03:01 ~ C " +
+      "~ BUS R2 0:04:41 0:05:01 ~ D " +
+      "~ Walk 20s " +
       "[0:01:11 0:05:31 4m20s 1tx $1510]";
 
     return RaptorModuleTestCase
       .of()
-      // TODO Investigate why this is not respecting the board/alight slack
-      //.add(TC_MIN_DURATION, PathUtils.withoutCost(expected))
-      .add(TC_MIN_DURATION_REV, PathUtils.withoutCost(expected))
+      // TODO: 2x10s alight slack is ignored in forward search for min-duration
+      .add(TC_MIN_DURATION, "[0:00 0:04 4m 1tx]")
+      // TODO: 2x30s Board- and 1x1m transfer slack is ignored in reverse search for min-duration
```

Comment:
When we board we know if there is a constrained transfer or not, we do not know witch one. But, this lett us add the board-slack when we know there isn´t. This should give a very small performance improvement. I have also done this for the *alight-slack*, but that require changes in the transit adaptor layer as well so I did not include it in this PR. Adding the `alight-slack` is rely the one that will impact performance, because that is the place when *board-* and *transfer-slack* is added in the reverse search.